sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -63,6 +65,117 @@ def test_input_estimator_unchanged():
     assert_true(transformer.estimator is est)
 
 
+def check_invalid_max_features(est, X, y):
```

Comment:
@qmaruf Why did you name each of these tests `check_*`? The scikit-learn standard is to name the unit tests with `test_*`.